### ISSUE TITLE
Fatal "unknown storage backend" when the backend is empty

### ISSUE THREAD

**Issue by wking** [openshift/cluster-image-registry-operator#58]
@RobertKrawitz reports this operator crashing on both libvirt and AWS with:

```
time=""2018-11-05T19:42:08Z"" level=info msg=""Cluster Image Registry Operator Version: 601f5c3-dirty""
time=""2018-11-05T19:42:08Z"" level=info msg=""Go Version: go1.10.3""
time=""2018-11-05T19:42:08Z"" level=info msg=""Go OS/Arch: linux/amd64""
time=""2018-11-05T19:42:08Z"" level=info msg=""operator-sdk Version: 0.0.6+git""
time=""2018-11-05T19:42:08Z"" level=info msg=""Metrics service cluster-image-registry-operator created""
time=""2018-11-05T19:42:08Z"" level=info msg=""generating registry custom resource""
time=""2018-11-05T19:42:08Z"" level=fatal msg=""unknown storage backend: """
```

From the first line of the logs, you can see that's with 601f5c32fdd327cb2e2309a8aee0b5362f0b892d from this repository.  I'll file a pull request at least expanding the logs for the missing config-map case.

**Comment by bparees** [comment]
this was fixed in https://github.com/openshift/cluster-image-registry-operator/pull/54

**Comment by wking** [comment]
> this was fixed in #54 

Ah, sorry.  I'd searched for "unknown storage backend", saw no hits, and didn't dig deeper :p.
